<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>DoctorIntr</class>
 <widget class="QWidget" name="DoctorIntr">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>240</width>
    <height>320</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>8</x>
     <y>20</y>
     <width>60</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>医生姓名</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>8</x>
     <y>60</y>
     <width>60</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>职称</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>8</x>
     <y>100</y>
     <width>60</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>科室</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="label_4">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>150</y>
     <width>60</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>坐诊时间</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="label_5">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>220</y>
     <width>60</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>从业经验</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
   </property>
  </widget>
  <widget class="QComboBox" name="placeComboBox">
   <property name="geometry">
    <rect>
     <x>80</x>
     <y>100</y>
     <width>120</width>
     <height>22</height>
    </rect>
   </property>
   <item>
    <property name="text">
     <string>心脏科</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>皮肤科</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>口科</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>鼻科</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>妇科</string>
    </property>
   </item>
  </widget>
  <widget class="QLineEdit" name="doctorNameLineEdit">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>20</y>
     <width>60</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="doctorWorkLineEdit">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>60</y>
     <width>60</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
  <widget class="QTextEdit" name="timeTextEdit">
   <property name="geometry">
    <rect>
     <x>80</x>
     <y>130</y>
     <width>120</width>
     <height>71</height>
    </rect>
   </property>
  </widget>
  <widget class="QTextEdit" name="exprienceTextEdit">
   <property name="geometry">
    <rect>
     <x>80</x>
     <y>210</y>
     <width>120</width>
     <height>71</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="doctorEnterPushButton">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>290</y>
     <width>51</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>存档</string>
   </property>
  </widget>
  <widget class="QPushButton" name="doctorReturnPushButton">
   <property name="geometry">
    <rect>
     <x>150</x>
     <y>290</y>
     <width>51</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>返回</string>
   </property>
  </widget>
  <widget class="QLabel" name="pictureLabel">
   <property name="geometry">
    <rect>
     <x>140</x>
     <y>20</y>
     <width>72</width>
     <height>72</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background:url(:/img/head_male_man_user_72px_1578_easyicon.net.png)</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
